### PR TITLE
add jump models for exp design

### PR DESCRIPTION
**Pull request by chriscoey**
want to add printing c,A,b,cones for the JuMP models to test whether any epsilons (see #233)

it seems that Pajarito results agree for the convex and jump models for A and E optimal formulations. but for D optimal, I just got different solutions for the experiments counts (of course, the objective is supposed to be different). this could indicate either a nonunique solution, or a bug in either D optimal model, or a Pajarito bug

### CODE REVIEW

**Comment by mlubin** [comment]
``JuMP.conicdata(m)`` returns ``c, A, b, var_cones, con_cones``

**Comment by mlubin** [comment]
It's time to stop speculating about correctness of the solution. Let's verify the optimal solution by enumeration.

**Comment by chriscoey** [comment]
I plan to do that yes, but first - JuMP has the same 4e-16 epsilons that Convex gave before

**Comment by mlubin** [comment]
Open an issue at JuMP with a simple, standalone example.

**Comment by chriscoey** [comment]
https://github.com/JuliaOpt/JuMP.jl/issues/972 
I'll update the example file again when that is implemented.

Ideally the formulation in conic form that JuMP yields would be as "efficient" as possible. Ie if I was to go and formulate these problems directly in MPB conic form (with svec SDP cones automatically enforcing symmetry), then what I get from JuMP would be the same. No equality constraints for all off-diagonals etc.

**Comment by mlubin** [comment]
> No equality constraints for all off-diagonals etc.

There shouldn't be any symmetry enforcing constraints in your revised models, so that's basically already the case.

File: examples/expdesign.jl [hunk 1]
```diff
@@ -81,16 +81,78 @@ dOpt = maximize(
...
+function geomean(dOpt, x::JuMP.Variable, y::JuMP.Variable) # SOCRotated
+    t = @variable(dOpt, lowerbound=0)
+    @constraint(dOpt, x*y >= t^2)
+    return t
+end
+
+dOpt = Model(solver=solver)
+np = @variable(dOpt, [j=1:p], Int, lowerbound=0, upperbound=nmax)
+@constraint(dOpt, sum(np) <= n)
+Q = @variable(dOpt, [1:q,1:q])
```
Comment (mlubin) [inline examples/expdesign.jl]: ``@variable(dOpt, [1:q,1:q], Symmetric)``

Comment (chriscoey) [inline examples/expdesign.jl]: OK does this reduce the number of variables then? rather than enforcing symmetry

Comment (mlubin) [inline examples/expdesign.jl]: Yes

File: examples/expdesign.jl [hunk 2]
```diff
@@ -81,16 +81,78 @@ dOpt = maximize(
...
+    t = @variable(dOpt, lowerbound=0)
+    @constraint(dOpt, x*y >= t^2)
+    return t
+end
+
+dOpt = Model(solver=solver)
+np = @variable(dOpt, [j=1:p], Int, lowerbound=0, upperbound=nmax)
+@constraint(dOpt, sum(np) <= n)
+Q = @variable(dOpt, [1:q,1:q], Symmetric)
+@constraint(dOpt, Q .== V * diagm(np./n) * V')
```
Comment (mlubin) [inline examples/expdesign.jl]: This should be updated to only enforce equality on the lower triangle.

Comment (chriscoey) [inline examples/expdesign.jl]: ok. well should i just wait for an implementation of the Symmetric SDconstraint in JuliaOpt/JuMP.jl#972 and then use that? that would result in an equally efficient formulation right?

Comment (mlubin) [inline examples/expdesign.jl]: You may be waiting for a while

**Pull request merged**